### DISCUSSION TITLE
Does this library support wear os?

### DISCUSSION THREAD

**Discussion by mdance** [patrykandpatrick/vico#284] [Questions]
Does this library support wear os devices like the pixel watch?

I am able to get a line chart to render, but the chart seems to overflow the screen.

I am very new to android development so hoping I'm just missing something simple...

Any pointers and gotchas would be greatly appreciated...

```
                    val model = entryModelOf(
                        1f,
                        2f,
                        3f,
                        4f,
                        5f,
                        4f,
                        3f,
                        2f,
                        1f,
                    )

                    val marker = rememberMarker()

                    Chart(
                        chart = lineChart(
                            lines = listOf(
                                lineSpec(
                                    lineColor = Primary,
                                    lineBackgroundShader = verticalGradient(
                                        arrayOf(
                                            Primary.copy(0.5f),
                                            Primary.copy(alpha = 0f)
                                        ),
                                    ),
                                ),
                            ),
                            persistentMarkers = remember(marker) {
                                mapOf(
                                    1f to marker,
                                    5f to marker,
                                )
                            }
                        ),
                        model = model,
                        marker = marker,
                        modifier = Modifier
                            .border(1.dp, Color.Red)
                    )
```

**Comment by mdance** [comment]
![WhatsApp Image 2023-06-14 at 22 06 53](https://github.com/patrykandpatrick/vico/assets/121057/f763c092-5528-47be-ad7e-91e260a8529d)

**Comment by Gowsky** [comment]
Hello @mdance, looks like this problem is a matter of adjusting the chart’s height and/or padding.

**Comment by mdance** [comment]
I still can't seem to get it to scale properly, it also seems like when I do set a width and height that its not scaling the data to fix the new dimensions and some of the datapoints are getting cropped off (the chart should have a marker at the minimum, and maximum price values which is working fine on the phone app)

```
        Box(
            contentAlignment = Alignment.CenterStart,
            modifier = Modifier
                .padding(10.dp)
                .border(
                    width = 1.dp,
                    color = Color.Red,
                )
                .padding(10.dp)
                .width(50.dp)
                .height(50.dp)
        ) {
            AssetChart(
                prices = prices,
            )
        }
```

The only modifier I have set in AssetChart is:

```
        modifier = modifier
            .fillMaxSize(),
```
Here is what the preview looks like

![Screenshot_2023-06-15_21-11-12](https://github.com/patrykandpatrick/vico/assets/121057/913fb670-68b9-43a7-89e3-1b3f603893b5)

**Comment by patrickmichalik** [comment]
The reason why the line is being cut off is that the chart is horizontally scrollable. Please try making the chart non-scrollable—the content will then be scaled down to fit the screen.
